modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -5,6 +5,10 @@
 @import 'image_picker';
 @import 'utilities';
 
+.ddm-field [hidden] {
+	display: none !important;
```

Comment:
Every time @marcoscv-work sees an `!important` he cries

![image](https://media.giphy.com/media/L95W4wv8nnb9K/giphy.gif)

Reply:
haha sorry 😅, this is from the implementation https://github.com/liferay/clay/pull/3534